metadata-ingestion/src/datahub/ingestion/source/s3/source.py

```diff
@@ -377,7 +377,7 @@ def read_file_spark(self, file: str, ext: str) -> Optional[DataFrame]:
                 ignoreLeadingWhiteSpace=True,
                 ignoreTrailingWhiteSpace=True,
             )
-        elif ext.endswith(".json"):
+        elif ext.endswith(".json") or ext.endswith(".jsonl"):
```

Comment:
have you tested/checked that spark.read.json() works with jsonl?

Reply:
Yes, It works fine without any additional option.
<img width="568" alt="image" src="https://github.com/datahub-project/datahub/assets/8328884/70f1b85b-6f81-4c70-9a68-d3035f904db8">